multimodal/src/autogluon/multimodal/utils/__init__.py

```diff
@@ -45,6 +45,7 @@
     COCODataset,
     bbox_xyxy_to_xywh,
     cocoeval,
+    convert_data_to_df,
```

Comment:
Just a small issue.

Since this function is only used in object detection. May be we should not move this `utils.convert_data_to_df`. Instead, we can keep it as `object_detection.utils.convert_data_to_df`.

Reply:
This has been fixed!